ffi/gn/.cargo/config

```diff
@@ -3,8 +3,3 @@
 # (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
 
 # AUTO-GENERATED in ffi.toml
-[source.crates-io]
-replace-with = "vendored-sources"
-
-[source.vendored-sources]
-directory = "vendor"
```

Comment:
There should be no changes in this file

Reply:
Ah not sure where this change come from. Removed now.